src/storage/config.rs

```diff
@@ -113,6 +109,15 @@ impl Config {
                     .into(),
             );
         };
+        let max_pool_size = std::cmp::max(1, SysQuota::cpu_cores_quota() as usize);
```

Comment:
Should be `std::cmp::max(1, std::cmp::min(4, cpu_num as usize))`

Reply:
updated. can be simplified to `std::cmp::max(4, cpu_num)`